<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QMainWindow" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="4">
     <widget class="QListView" name="right_list_view"/>
    </item>
    <item row="1" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QListView" name="left_list_view"/>
    </item>
    <item row="1" column="3">
     <widget class="QTreeView" name="right_tree_view"/>
    </item>
    <item row="1" column="0">
     <widget class="QTreeView" name="left_tree_view"/>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QLineEdit" name="rightAdress"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLineEdit" name="leftAdress"/>
    </item>
   </layout>
   <zorder>left_list_view</zorder>
   <zorder>right_list_view</zorder>
   <zorder>left_tree_view</zorder>
   <zorder>right_tree_view</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>leftAdress</zorder>
   <zorder>rightAdress</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuHide_trees">
     <property name="title">
      <string>Hide trees</string>
     </property>
     <addaction name="actionHide_both"/>
     <addaction name="actionHide_left_tree"/>
     <addaction name="actionHide_right_tree"/>
    </widget>
    <widget class="QMenu" name="menuShow_trees">
     <property name="title">
      <string>Show trees</string>
     </property>
     <addaction name="actionShow_both"/>
     <addaction name="actionShow_left_tree"/>
     <addaction name="actionShow_right_tree"/>
    </widget>
    <addaction name="menuHide_trees"/>
    <addaction name="menuShow_trees"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_program"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_program">
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="actionHide_left_tree">
   <property name="text">
    <string>Hide left tree</string>
   </property>
  </action>
  <action name="actionHide_right_tree">
   <property name="text">
    <string>Hide right tree</string>
   </property>
  </action>
  <action name="actionShow_left_tree">
   <property name="text">
    <string>Show left tree</string>
   </property>
  </action>
  <action name="actionShow_right_tree">
   <property name="text">
    <string>Show right tree</string>
   </property>
  </action>
  <action name="actionHide_both">
   <property name="text">
    <string>Hide both</string>
   </property>
  </action>
  <action name="actionShow_both">
   <property name="text">
    <string>Show both</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
